_config.yml

```diff
@@ -20,6 +20,7 @@ sass:
 exclude:
   # top-level log and markdown files
   - /*.log
+  - /*.md
```

Comment:
It looks like the state pages subpages (e.g. `explore/WY/`) aren't building on federalist. I'm assuming that this is why.